libs/shared/util-blockchain/chains/index.ts

```diff
@@ -0,0 +1,32 @@
+import chains from './chains.json';
+
+export const getChain = (chainId: number) =>
+  chains.find((c) => c.chainId === chainId);
+
+// Not all chains are relevant for PieDAO products, you can define the priority ones here
+const SUPPORTED_CHAINS = {
+  ETHEREUM: 1,
+  POLYGON: 137,
+  FANTOM: 250,
+  AVALANCE: 41334,
+  OPTIMISM: 11,
+  ARBITRUM: 42161,
+  GNOSIS: 100,
+};
```

Comment:
you should add `as const` at the end. This will be useful later when you use `SUPPORTED_CHAINS` in a type guard for example.

Reply:
Agree with that change.

Reply:
Actioned